docs/source/platforms/agents/running-endpoints.rst

```diff
@@ -0,0 +1,117 @@
+.. _Platforms-Agents-Running-Endpoints:
+
+==================================
+Platforms Agents Running Endpoints
+==================================
+
+Platforms Agents Running endpoints expose functionality associated with the running status of agents on the platform.
+This includes determining whether an agent is running as well as starting and stopping the agent.
+
+.. attention::
+    All Platforms Agents Running endpoints require a JWT bearer token obtained through the
+    ``POST /authenticate`` or ``PUT /authenticate`` endpoints.
+
+--------------
+
+GET /platforms/:platform/agents/:agent/running
+==============================================
+
+Retrieve the running status of the specified agent.
+
+Request:
+--------
+
+    -  Authorization: ``BEARER <jwt_access_token>``
+
+Response:
+---------
+
+    *  **With valid BEARER token on success:** ``200 OK``
+        - Content Type: application/json
+        - Body:
+
+          .. code-block:: json
+
+                {
+                    "status": true|false
```

Comment:
The value of "status" doesn't display right in the generated html. Please see if the | symbol can be escape for rst to be happy